sig-cloud/README.md

```diff
@@ -56,9 +56,9 @@ _OpsMx_
 
 
 ### Meeting Cadence
- - 1 hour, monthly, on **[Zoom](TODO-need-zoom-link)**
+ - 1 hour, monthly, on **[Google Meet](https://8x8.vc/greencapsdesirepolitely)**
```

Comment:
This doesn't look like a google meet link, but that probably doesn't matter.

Reply:
```suggestion
 - 1 hour, monthly, on **[Google Meet](https://meet.google.com/afb-rppx-zye)**
```

Reply:
Ah yea, I think Google Meet uses 8x8 as the backend now, so the link's are basically the same thing, with different frontends.